undertow/src/main/java/org/wildfly/extension/undertow/ActiveRequestTrackerService.java

```diff
@@ -0,0 +1,56 @@
+package org.wildfly.extension.undertow;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import io.undertow.predicate.Predicate;
+import io.undertow.server.HttpHandler;
+import io.undertow.server.handlers.ActiveRequestTrackerHandler;
+import org.jboss.msc.service.Service;
+import org.jboss.msc.service.StartContext;
+import org.jboss.msc.service.StartException;
+import org.jboss.msc.service.StopContext;
+
+public class ActiveRequestTrackerService implements Service<ActiveRequestTrackerService> {
```

Comment:
Exposing the MSC Service API via the service value is an anti-pattern, i.e. a consumer should not have access to the start(...)/stop(...) methods.
Since this service seems to provide a ActiveRequestTrackerHandler, perhaps you mean to use Service<ActiveRequestTrackerHandler> instead?

Reply:
TBH, I did here what I saw in AccessLogService: undertow/src/main/java/org/wildfly/extension/undertow/AccessLogService.java

Reply:
See https://issues.redhat.com/browse/WFLY-8512

Reply:
I think I have this cleaned up as requested locally. I'll ping you when the PR is updated.